templates/k8s-ansible-service-broker.yaml.j2

```diff
@@ -0,0 +1,241 @@
+#
+# Limitations With the Kubernetes template
+# ----------------------------------------
+# - ssl
+#
```

Comment:
I would add that the broker is not included in this template.

Reply:
the broker is down on line 193